### ISSUE TITLE
SingleLinePerProperty gets confused with nesting

### ISSUE THREAD

**Issue by lencioni** [brigade/scss-lint#709]
With the following configuration:

```yaml
SingleLinePerProperty:
  enabled: true
  allow_single_line_rule_sets: false
```

And the following SCSS:

```scss
.test { padding: 0; margin: 0; }
```

The SingleLinePerProperty correctly reports a lint here. However, if this is nested:

```scss
.testing {
  .test { padding: 0; margin: 0; }
}
```

it is quiet.
